lib/topology/platforms/base.py

```diff
@@ -439,5 +439,20 @@ def disable(self):
         """
         self._enabled = False
 
+    def _log_command(self, command):
+        """
+        Command logging function for low-level shell API usage.
+
+        FIXME: how can we specify the shell here?
+        """
+        print('{} [{}].send_command(\'{}\', shell=\'{}\') ::'.format(
```

Comment:
Should we use the ``[node_name].send_command(...)`` syntax here? In these cases is the shell object ``send_command`` method that is executed, not the node one.